### ISSUE TITLE
Jekyll 3.0 points out Liquid rendering error in posts.html

### ISSUE THREAD

**Issue by jeremymodell** [mmistakes/so-simple-theme#141]
Edit: Error fixed, see explanation.
So I upgraded to the newest jekyll beta (3.0 beta 2) and the good news is that most things work, except for a quick fix needed in posts.html. After upgrading while trying to generate the site with build jekyll i encountered a weird error for liquid  
``` Generating... 
 Liquid Exception: Liquid syntax error: Expected dotdot but found id in "(site.owner.disqus-shortname and page.comments) or site.comments" in _layouts/post.html
jekyll 3.0.0.pre.beta2 | Error:  Liquid syntax error: Expected dotdot but found id in "(site.owner.disqus-shortname and page.comments) or site.comments" 
```
and after looking at _layout/posts.html and searching the issue online it seems that the error always existed, parentheses are not permitted  by Liquid but for the first time it will prevent a build of a site because Jekyll 3 will show the error. So to fix the issue we just remove the offending parentheses in the disqus part of of post.html starting at line 39 like so 
``` html 
{% if site.owner.disqus-shortname and page.comments or site.comments %}<span class="entry-comments"><i class="fa fa-comment-o"></i> <a href="#disqus_thread">Comment</a></span>{% endif %}
        {% if page.share %}{% include social-share.html %}{% endif %}
        {% if page.ads == true %}{% include ad-sidebar.html %}<!-- /.google-ads -->{% endif %}
      </footer>
      <div class="entry-content">
        {{ content }}
        {% if site.owner.disqus-shortname and page.comments or site.comments %}
          <div id="disqus_thread"></div><!-- /#disqus_thread -->
          {% include disqus-comments.html %}
```

**Comment by mmistakes** [comment]
Seems valid. Care to submit a PR so I can test it out. Just want to make sure the Disqus script isn't loading if those conditions aren't met. In the past its mistakenly loaded on pages even when someone hadn't included a shortname in their `_config.yml` or `comments: true` in a post. 

The bit with the parentheses was submitted as a PR if I recalled correctly to cover cases where someone might want comments to appear on all posts without having to manually add `comments: true` to them. Seemed to work at the time without errors so I left it alone. Definitely want things to work with the upcoming 3.0 release.

**Comment by mmistakes** [comment]
Should be fixed as of [2.3.2](https://github.com/mmistakes/so-simple-theme/releases/tag/2.3.2)
